src/main/java/com/verily/genomewarp/utils/GenomeRangeUtils.java

```diff
@@ -29,21 +29,19 @@
   private static final Logger logger = Logger.getLogger(GenomeRangeUtils.class.getName());
 
   /**
-   * Splits a given set of genome ranges at non-DNA characters
+   * Returns a map from chromosome to list of ranges present in the BED filtered by queryChrom.
    *
-   * <p> This function reads in ranges using a BufferedReader and splits
-   * the given range along non-strict DNA characters (non ACTGactg).
    *
-   * @param refFasta the reference genome, containing the string bases
    * @param bed a BufferedReader containing the range reads
    * @param queryChroms the set of query chromosomes to include
-   * @return a hashmap of chromosome to arraylist of GenomeRanges only containing valid DNA characters
+   * @return a hashmap of chromosome to List of GenomeRanges on the requested chromosomes.
+   * @throws IllegalArgumentException if the BED does not have all ranges from a single chromosome
+   *   together.
```

Comment:
s/together/together in the same order

Reply:
That's not actually true in this function, which only checks for all ranges from each chromosome to be in a contiguous chunk in the file. The within-chromosome sorting is checked in the {ensure,create}SortedMergedRanges functions.